app/models/document.rb

```diff
@@ -1,7 +1,7 @@
 class Document
   attr_reader :title, :public_timestamp, :is_historic, :government_name,
               :content_purpose_supergroup, :document_type, :organisations,
-              :release_timestamp, :es_score
+              :release_timestamp, :es_score, :top_result, :description
```

Comment:
Why has description been made public?

Reply:
We need to add the description to the top result, therefore needed it publicly available to use in ResultSetPresenter.

Reply:
Maybe add that to the commit message?

Reply:
I think finder frontend exposes `summary` which is usually the underlying document description.
https://github.com/alphagov/finder-frontend/blob/05a65874c6633787f487cebdaa7af8ad7d71bad3/app/models/document.rb#L35

Reply:
The summary always comes back with null so I had to use the document description